common/scala/src/main/scala/whisk/core/connector/Message.scala

```diff
@@ -88,7 +88,8 @@ object ActivationMessage extends DefaultJsonProtocol {
  */
 case class CompletionMessage(
     override val transid: TransactionId,
-    response: WhiskActivation)
+    response: WhiskActivation,
+    invoker: String)
```

Comment:
we do know when the request is posted which invoker it was assigned to.
it may be necessary anyway to keep this field in the active ack since in the future another invoker may steal a request.